counterparty-core/counterpartycore/lib/messages/issuance.py

```diff
@@ -847,10 +848,10 @@ def unpack(db, message, message_type_id, block_index, return_dict=False):
             "asset": asset,
             "subasset_longname": subasset_longname,
             "quantity": quantity,
-            "divisible": divisible,
-            "lock": lock,
-            "reset": reset,
-            "callable": callable_,
+            "divisible": bool(divisible),
+            "lock": bool(lock),
```

Comment:
So this is a breaking API change... is it crazy to bump the version number for this?

Reply:
I guess it's just for this event... that's very minor.

Reply:
Was this a recent regression?